app/controllers/theme_controller.py

```diff
@@ -0,0 +1,158 @@
+import json
+from pathlib import Path
+from typing import Optional
+
+from loguru import logger
+
+from app.utils.app_info import AppInfo
+
+
+class ThemeController:
+    def __init__(self, default_theme: str = "RimPy") -> None:
+        self.app_info = AppInfo()
+        logger.info("Initializing ThemeController")
+        self.default_theme = default_theme
+        self._theme_name = self.load_theme_name() or self.default_theme
+        logger.info(f"Current theme name: {self._theme_name}")
+        self.themes = self.get_supported_themes()
+        logger.info(f"Supported themes: {self.themes}")
+        self.theme_stylesheets_folder = [
+            self.app_info.theme_data_folder,
+            self.app_info.theme_storage_folder,
```

Comment:
Flip the order to prioritize storage when looking for themes for the case where there are duplicate names.

Reply:
yeah right done Thanks